action.yml

```diff
@@ -36,4 +36,6 @@ runs:
     ARG_DEBUG: ${{ inputs.debug }}
   entrypoint: '/scripts/lock.sh'
   post-entrypoint: '/scripts/unlock.sh'
+  if: ${{ always() }}
```

Comment:
I only see documented `post-if` https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#post-entrypoint

Maybe we have to remove this line

Reply:
And it seems it handles it well enough: https://github.com/ben-z/gh-action-mutex/blob/bc97a5192de56202e1b5d1d3eeca2ecbbef8523c/rootfs/scripts/utils.sh#L101-L105